modules/k8s-nmstate-deploying-nmstate-CLI.adoc

```diff
@@ -84,19 +99,26 @@ metadata:
 EOF
 ----
 
-.Verification
-
-* Confirm that the deployment for the `nmstate` operator is running:
+. Verify the pods for NMState Operator are running:
```

Comment:
Here the Operator is prepended with "NMState" while elsewhere it's `nmstate`. Best to stick with one format for consistency.

Reply:
Here I'm referring to the Operator in a general sense like it is done in the introductory sentence; whereas, in other spots `nmstate` is referring to the Operators namespace or other fields. In short, I was trying to match the consistency of the current docs, which switch between the two.

Reply:
Ah, thanks for clarifying.